### ISSUE TITLE
Support clicking the service name in the chart to link to the tracking page.

### ISSUE THREAD

**Issue by heihaozi** [apache/skywalking-rocketbot-ui#529]
There will be the name of the service in the chart, as shown below:

![OneMore](https://user-images.githubusercontent.com/2956927/129997596-50f5ba11-2d52-44a9-b83b-64d2253a9b18.png)

If users click on the service name, they can link to the corresponding trace page, it will be more convenient for the user to use.

**Comment by wu-sheng** [comment]
How could do you this is a service name?

**Comment by wu-sheng** [comment]
Top N doesn't have to be service, it could be instance and endpoint.

**Comment by wu-sheng** [comment]
If you want to do this, you have to make the backend could response the entity type of metric name, which doesn't exist.

**Comment by heihaozi** [comment]
Yes, we need to determine the entity type and metricName, and only those that meet the requirements can be linked.

**Comment by wu-sheng** [comment]
I think you should discuss to get the details before running into the codes. If we want to do this, we definitely need to figure out how to enhance the backend first.

**Comment by heihaozi** [comment]
We don't need to enhance the backend, there are already entity type and metric name attributes in the chart component of UI.
